resources/TrialApp.didact.md

```diff
@@ -0,0 +1,55 @@
+# Welcome to your Trial App Didact Tutorial
+
+This guide will show you how to make and deploy changes to your Azure App Service Trial App. Your trial will last 60 minutes. You can find more information about Azure App Service [here](https://docs.microsoft.com/en-us/azure/app-service/).
+
+## Getting Started
+
+### Step 1: Clone the trial app source code
+
+1. Locate your imported Trial App in the Azure view.
+
+<img src="./trialAppTutorial/viewTrialAppScreenshot.png" style="width: 256px;">
+
+2. Right click on the Trial App.
+3. Select `clone`.
+
+<img src="./trialAppTutorial/rightClickCloneScreenshot.png" style="width: 256px;">
+
+4. Choose a folder to clone the Trial App source code to.
+5. When the repository has finished cloning a notification like the one below will appear. Open the folder in VS Code or add it to the workspace.
+
+<img src="./trialAppTutorial/cloneNotificationScreenshot.png" style="width: 256px;">
+
+### Step 2: Make some changes to your app
+
+Make some changes to your trial app. Try changing some text in `template.html`.
```

Comment:
Might be overboard, but maybe we can add a screenshot of one of your changes?  I'm not sure what level of experience we're expecting from the users...

Reply:
I added examples instead. So the user could copy and paste if they want.